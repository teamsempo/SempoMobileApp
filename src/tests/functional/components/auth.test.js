import React from 'react';
import { shallow } from 'enzyme';
import PinComponent from '../../../components/auth/GenericPinComponent.js';
import configureMockStore from 'redux-mock-store';

describe('PinComponent', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const mockStore = configureMockStore([]);
            const component = shallow(<PinComponent store={mockStore} promptText={'Prompt Text'}/>).dive();
            expect(component).toMatchSnapshot();
        });
    });
});

