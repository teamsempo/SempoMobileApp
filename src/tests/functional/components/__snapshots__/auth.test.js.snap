// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PinComponent Rendering should match to snapshot 1`] = `
<GenericPinComponent
  buttonBgColor="#FFF"
  buttonTextColor="#333"
  clear={false}
  deleteText="DEL"
  disabled={false}
  inputActiveBgColor="#333"
  inputBgColor="#333"
  inputBgOpacity={0.1}
  login={false}
  loginRequest={[Function]}
  otp={null}
  phone={null}
  pinLength={4}
  promptText="Prompt Text"
  region={null}
  returnType="string"
  set={false}
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
  verify={true}
/>
`;
